examples/build.gradle

```diff
@@ -1,16 +1,193 @@
 description = 'OpenCensus Examples'
 
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.4.2'
+        classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
+        classpath "net.ltgt.gradle:gradle-apt-plugin:0.10"
+        classpath 'com.github.ben-manes:gradle-versions-plugin:0.15.0'
+        classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.6"
+    }
+}
+
+// Display the version report using: ./gradlew dependencyUpdates
+// Also see https://github.com/ben-manes/gradle-versions-plugin.
+apply plugin: 'com.github.ben-manes.versions'
+apply plugin: "checkstyle"
+apply plugin: 'maven'
+apply plugin: 'idea'
+apply plugin: 'java'
+apply plugin: "signing"
+apply plugin: "jacoco"
+// The plugin only has an effect if a signature is specified
+apply plugin: 'ru.vyarus.animalsniffer'
+apply plugin: 'findbugs'
+apply plugin: 'net.ltgt.apt'
+// Plugins that require java8
+if (JavaVersion.current().isJava8Compatible()) {
+    apply plugin: "net.ltgt.errorprone"
+    apply plugin: 'com.github.sherter.google-java-format'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+group = "io.opencensus"
+version = "0.11.0-SNAPSHOT"
+
+// change to the version you want to use.
+def opencensusVersion = "0.11.0-SNAPSHOT" // CURRENT_OPENCENSUS_VERSION
```

Comment:
I think we should just use the latest released version here, like we do in the example dependency declarations in the readme.

Reply:
We can do that but then you cannot add example for an API that is not released yet. Maybe not a big deal

Reply:
Are you suggesting using `version` only?

My logic here is that the example project itself has a release version (defined by `version`), while it could be built against other version of opencensus api/impl (defined by `opencensusVersion`, and these dependencies will be downloaded from maven instead of referencing the project itself).

Reply:
you can ask users to do a "./gradle install" for the main project, which builds and install locally the main project. I am fine with using the latest release.